components/maker/payment/helpers.js

```diff
@@ -0,0 +1,21 @@
+import { address, networks } from 'bitcoinjs-lib'
+import * as c32Check from 'c32check'
+import memoize from 'lodash/memoize'
+
+export const validateBTC = memoize(addr => {
+  try {
+    address.toOutputScript(addr, networks.bitcoin)
+    return true
+  } catch (error) {
+    return false
+  }
+})
+
+export const validateSTX = memoize(addr => {
+  try {
+    c32Check.c32addressDecode(addr)
+    return true
+  } catch (error) {
+    return false
+  }
+})
```

Comment:
These exist within the package `@blockstack/stacks-utils` -> https://github.com/blockstack/stacks-utils/tree/feature/add-data-fetches-and-hardware-wallet-support

Reply:
Good to know, thanks.

Reply:
Ran into issue using this lib, created issue here https://github.com/blockstack/stacks-utils/issues/4